standard/links_stream.lua

```diff
@@ -11,53 +11,50 @@ Module containing utility functions for streaming platforms.
 ]]
 local StreamLinks = {}
 
+local Array = require('Module:Array')
 local Class = require('Module:Class')
 local Logic = require('Module:Logic')
+local Page = require('Module:Page')
 local String = require('Module:StringUtils')
+local Table = require('Module:Table')
 local Variables = require('Module:Variables')
 
---[[
-List of streaming platforms supported in Module:Countdown. This is a subset of
-the list in Module:Links
-]]
+local TLNET_STREAM = 'stream'
+
+--List of streaming platforms supported in Module:Countdown.
 StreamLinks.countdownPlatformNames = {
 	'afreeca',
-	'afreecatv',
 	'bilibili',
-	'cc163',
+	'cc',
 	'dailymotion',
 	'douyu',
 	'facebook',
-	'huomao',
-	'huya',
+	'huomaotv',
```

Comment:
Would it not be easier to change the icon-name (`LP-XXX` css class) instead of input values on all wikis?

Reply:
cc163 was always bad imo as it has a number at the end, infobox and most other modules use plain `cc` anyways
adding nimo as alias (infoboxes use nimotv)
huya & huomao already have both in the css

Reply:
Will you take care of any parameter renaming that's needed?

Reply:
added an alias into countdown module
checking on the in total 14 usages (all on warcraft) currently

Reply:
there is a total of 1 conten page using it according to https://tools.liquipedia.space/?view=search